railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_1.rb.tt

```diff
@@ -46,3 +46,18 @@
 
 # Make `form_with` generate non-remote forms by default.
 # Rails.application.config.action_view.form_with_generates_remote_forms = false
+
+# Set the default queue name for the analysis job to the queue adapter default.
+# Rails.application.config.active_storage.queues.analysis = nil
+
+# Set the default queue name for the purge job to the queue adapter default.
+# Rails.application.config.active_storage.queues.purge = nil
+
+# Set the default queue name for the incineration job to the queue adapter default.
+# Rails.application.config.action_mailbox.queues.incineration = nil
+
+# Set the default queue name for the routing job to the queue adapter default.
+# Rails.application.config.action_mailbox.queues.routing = nil
+
+# Set the default queue name for the mail deliver job to the queue adapter default.
+# Rails.application.config.action_mailer.deliver_later_queue_name = nil
```

Comment:
Apparently this should be
```ruby
ActionMailer::Base.deliver_later_queue_name = nil
```
because as said in https://github.com/rails/rails/issues/39810 "Since your initializer is only loaded after the framework, setting `config.action_mailer.deliver_later_queue_name` will not have any effect."

I tried to create a PR but I don't know which branch/tag I should start on to make a change for this file.